packages/config/src/validation.ts

```diff
@@ -33,7 +33,7 @@ const _isFullyQualifiedUrl = (value: any): ErrResult | boolean => {
   return _.isString(value) && /^https?\:\/\//.test(value)
 }
 
-const isArrayOfStrings = (value: any): ErrResult | boolean => {
+export const isArrayOfStrings = (value: any): ErrResult | boolean => {
   return _.isArray(value) && _.every(value, _.isString)
```

Comment:
Same comment as above, this isn't currently wired up to return the correct error response for logging if this fails, it needs to uses to return `errMsg` if this condition fails. You'll need to check to ensure this doesn't impact `isStringOrArrayOfStrings`.

Reply:
Ah that's starting to make a lot more sense! I think what I might do then to lessen the impact is just create a `isNullOrArrayOfStrings` that returns `errMsg` or `true` similar to `isStringOrArrayOfStrings`. That way things are at least way more consistent with what we have.

Reply:
fixed in dccf967ed9